lib/src/notification/notification.component.scss

```diff
@@ -0,0 +1,286 @@
+@import "~@sebgroup/bootstrap/scss/variables";
+
+$small: 12px;
+
+@mixin apply-theme($bg-color, $close-hover) {
+    .content-wrapper {
+        background-color: $bg-color;
+
+        &.clickable {
+            &:hover {
+                background-color: lighten($bg-color, 8%);
+            }
+
+            &:active {
+                background-color: $bg-color;
+            }
+        }
+
+        .actions-wrapper {
+            .action-wrapper {
+                .notification-action {
+                    &:hover {
+                        color: $bg-color;
+                    }
+                }
+            }
+        }
+    }
+
+    .dismiss-btn {
+        svg {
+            &:hover {
+                fill: $close-hover;
+            }
+        }
+    }
+}
+
+.custom-notification {
+    position: fixed;
+    display: block;
+    z-index: 9999;
```

Comment:
can you use the `z-index` variable from bootstrap? I think there is one defined for `toast` component or something similar, if not, just use the highest one which I think is the `tooltip`